src/services/__tests__/AsyncStorage.spec.ts

```diff
@@ -48,7 +52,6 @@ describe('AsyncStorage', () => {
 
   describe('ExerciseProgress', () => {
     beforeEach(() => {
```

Comment:
Is this really also needed? As the "parent" describe also has a beforeEach which clears the storage?